mls-rs-provider-sqlite/src/lib.rs

```diff
@@ -54,13 +54,39 @@ impl mls_rs_core::error::IntoAnyError for SqLiteDataStorageError {
     }
 }
 
+#[derive(Clone, Debug)]
+pub enum JournalMode {
+    DELETE,
+    TRUNCATE,
+    PERSIST,
+    MEMORY,
+    WAL,
+    OFF
+}
+
+// Note: for in-memory dbs (such as what the tests use), the only available options are MEMORY or OFF
+// Invalid modes do not error, only no-op
```

Comment:
Would it make sense to make this a doc comment (`///`) and attach it to the definition of `JournalMode` so that it shows up in the documentation?